go.mod

```diff
@@ -9,14 +9,12 @@ require (
 	github.com/hashicorp/golang-lru v0.5.3 // indirect
 	github.com/pkg/errors v0.8.1 // indirect
 	golang.org/x/crypto v0.0.0-20190923035154-9ee001bba392 // indirect
-	golang.org/x/net v0.0.0-20190923162816-aa69164e4478 // indirect
 	golang.org/x/sys v0.0.0-20190924154521-2837fb4f24fe // indirect
 	golang.org/x/time v0.0.0-20190921001708-c4c64cad1fd0 // indirect
 	google.golang.org/appengine v1.6.5 // indirect
 	google.golang.org/genproto v0.0.0-20190916214212-f660b8655731 // indirect
 	gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15 // indirect
-	gopkg.in/inf.v0 v0.9.1 // indirect
-	istio.io/api v0.0.0-20191109011911-e51134872853
-	k8s.io/apimachinery v0.0.0-20191004115801-a2eda9f80ab8
+	istio.io/api v0.0.0-20200106191429-c785926a849c
+	k8s.io/apimachinery v0.17.0
 	k8s.io/client-go v0.0.0-20191016111102-bec269661e48
```

Comment:
use v0.17.0?

Reply:
k8s client-go changed the versioning name for their repos: https://github.com/kubernetes/client-go#client-go

Reply:
istio/istio uses v0.17.0. I think the way you have it now is the old way

Reply:
k8s.io/client-go is modified to use the new way.

Reply:
nice, thanks! that will make consumption much easier probably